src/command.py

```diff
@@ -49,6 +49,43 @@ def __repr__(self):
                f"Gem message: {self.gem_message}"
 
 
+class GemsCounterFromMessage:
+    def __get_int_converted_gem_count(self, gem_count_string: str):
+        if gem_count_string.isdigit():
+            return int(gem_count_string)
+        try:
+            gem_count_float = float(gem_count_string)
```

Comment:
To me this feels like it should just fail instead of trying to convert to float then int?
```python
try:
    return int(gem_count_string)
except some_error:
    return 0
```

Reply:
yes, i had considered if we return 0 instead of int value of the float. according to the latest push, it returns 0. test modified accordingly.